db/src/main/resources/db/migration/V1_028__add_phone_number_to_claimant.sql

```diff
@@ -0,0 +1 @@
+alter table claimant add column phone_number varchar(13);
```

Comment:
Should this be `NOT NULL`, or is that happening in another PR?

Reply:
That'll be in the next PR as adding notnull now will cause the tests to fail. A TODO has been added to ClaimantTest linking to the next subtask.